doc/gemini_gmos.rst

```diff
@@ -18,6 +18,23 @@ taken in Nod+Shuffle mode by simply replicating the calibrations.
 That is, we do *not* subtract the nodded images but reduce
 it as if it were a separate slit.
 
+For this mode, you need to specify the `gemini_gmos_north_ham_ns` 
+spectrograph.
+
+Slits
```

Comment:
Are these the defaults in the spectrograph file for this instrument? If you do that, then it is not necessary to add to the pypeit file.

Reply:
These are not defaults and should not be as GMOS can have a slit mask.

We could try to make them defaults for the longslit if we can reliably
tell when it *is* the longslit.

Reply:
Okay that is good. Please update the dev suite examples accordingly if there are any and add a comment explaining these lines.